src/euphorie/client/model.py

```diff
@@ -1206,6 +1206,16 @@ def completion_percentage(self):
         return completion_percentage
 
 
+class SessionRedirect(BaseObject):
+    """Mapping of old deleted sessions to their new rebuilt counterparts"""
+
+    __tablename__ = "session_redirect"
+
+    id = schema.Column(types.Integer(), primary_key=True, autoincrement=True)
```

Comment:
Do we need the `id` here?
I would say that the primary key is the `old_session_id`

Reply:
When I tried to make the `old_session_id` the primary key (without `autoincrement`) I got the error

```
sqlalchemy.exc.ArgumentError: Mapper mapped class SessionRedirect->session_redirect could not assemble any primary key columns for mapped table 'session_redirect'
```

so I added a separate primary key.

Reply:
I can look into it again if you think it's worth it.

Reply:
Actually it works with `old_session_id`. Not sure what caused the error previously, but it must have been something else.